frontend/server/src/DAO/Runs.php

```diff
@@ -255,7 +255,22 @@ final public static function getAllRuns(
                     )
                 LIMIT 1
                 ) AS
-                    execution
+                    execution,
+                ( SELECT
+					IF(
+						verdict IN ("JE", "CE"), "MEMORY_NOT_AVAILABLE",
+					IF(
+						verdict IN ("ML", "MLE"), "MEMORY_EXCEEDED", "MEMORY_AVAILABLE"
+					))
```

Comment:
```suggestion
                    IF(
                        verdict IN ("JE", "CE"), "MEMORY_NOT_AVAILABLE",
                    IF(
                        verdict IN ("ML", "MLE"), "MEMORY_EXCEEDED", "MEMORY_AVAILABLE"
                    ))
```

se pueden usar espacios por consistencia?